chia/rpc/wallet_rpc_api.py

```diff
@@ -999,6 +1001,15 @@ async def send_notification(self, request) -> EndpointResult:
         await self.service.wallet_state_manager.add_pending_transaction(tx)
         return {"tx": tx.to_json_dict_convenience(self.service.config)}
 
+    async def sign_message(self, request) -> EndpointResult:
+        puzzle_hash: bytes32 = decode_puzzle_hash(request["address"])
+        hex_message = request["message"]
+        if hex_message.startswith("0x") or hex_message.startswith("0X"):
```

Comment:
I think we have a bytes32.fromhex method

Reply:
hex_message is not a bytes32

Reply:
We also have chia.util.byte_types.hexstr_from_bytes